commercetools-models/src/main/java/io/sphere/sdk/products/queries/ProductByIdHeadImpl.java

```diff
@@ -0,0 +1,36 @@
+package io.sphere.sdk.products.queries;
+
+import io.sphere.sdk.http.NameValuePair;
+import io.sphere.sdk.products.Product;
+import io.sphere.sdk.products.expansion.ProductExpansionModel;
+import io.sphere.sdk.products.search.PriceSelection;
+import io.sphere.sdk.queries.MetaModelHeadDslBuilder;
+import io.sphere.sdk.queries.MetaModelHeadDslImpl;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static io.sphere.sdk.products.search.PriceSelectionQueryParameters.extractPriceSelectionFromHttpQueryParameters;
+import static io.sphere.sdk.products.search.PriceSelectionQueryParameters.getQueryParametersWithPriceSelection;
+
+final class ProductByIdHeadImpl extends MetaModelHeadDslImpl<Product, Product, ProductByIdHead, ProductExpansionModel<Product>> implements ProductByIdHead {
```

Comment:
The Head methods don't have expansion, price selection etc

Reply:
thanks, I followed the wrong PR about the price selection